src/xlsx/helpers/content_helpers.ts

```diff
@@ -72,12 +72,8 @@ export function getCellType(value: number | string | boolean): string {
   }
 }
 
-/**
- * For some reason, Excel will only take the devicePixelRatio (i.e. interface scale on Windows desktop)
- * into account for the height.
- */
 export function convertHeightToExcel(height: number): number {
-  return Math.round(HEIGHT_FACTOR * height * window.devicePixelRatio * 100) / 100;
+  return Math.round(HEIGHT_FACTOR * height * 100) / 100;
```

Comment:
> why do you need to remove this?

@LucasLefevre It's about keeping image aspect ratio (because it depends on the ratio of row height and col width).
Before removal, we only multiply this ratio when converting height to excel data (exporting), not vice versa (importing), so it causes the inconsistency. Besides that, we only did it for row height and not for col width, so when you opened the exported .xlsx file you could see the ratio of row heights and col widths is not as we saw in o-spreadsheet. 
I also tried multiplying `devicePixelRatio` on col width too, but except noticing the cell dimensions become larger than what we saw in o-spreadsheet before exporting, I didn't notice anything else, so I deleted it. 🙂